<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Reporter log started]]>
    </line>
    <line>
      <![CDATA[Login was successful. Moving to next step]]>
    </line>
  </reporter-output>
  <suite started-at="2020-05-13T04:56:40Z" name="preserve-order Suite" finished-at="2020-05-13T04:57:02Z" duration-ms="22660">
    <groups>
    </groups>
    <test started-at="2020-05-13T04:56:40Z" name="tests" finished-at="2020-05-13T04:57:02Z" duration-ms="22660">
      <class name="Reusable_BE.TC_Mobile_New_Modify_CancelBooking">
        <test-method is-config="true" signature="deleteLog()[pri:0, instance:Reusable_BE.TC_Mobile_New_Modify_CancelBooking@10959ece]" started-at="2020-05-13T10:26:40Z" name="deleteLog" finished-at="2020-05-13T10:26:40Z" duration-ms="10" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteLog -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:Reusable_BE.TC_Mobile_New_Modify_CancelBooking@10959ece]" started-at="2020-05-13T10:26:40Z" name="setUp" finished-at="2020-05-13T10:26:40Z" duration-ms="232" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="Beforem(java.lang.String)[pri:0, instance:Reusable_BE.TC_Mobile_New_Modify_CancelBooking@10959ece]" started-at="2020-05-13T10:26:40Z" name="Beforem" finished-at="2020-05-13T10:26:51Z" duration-ms="10863" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Reporter log started]]>
            </line>
            <line>
              <![CDATA[Login was successful. Moving to next step]]>
            </line>
          </reporter-output>
        </test-method> <!-- Beforem -->
        <test-method signature="Flow(java.util.HashMap)[pri:0, instance:Reusable_BE.TC_Mobile_New_Modify_CancelBooking@10959ece]" started-at="2020-05-13T10:26:52Z" name="Flow" description="Booking Room in Booking Engine" data-provider="MobileBooking" finished-at="2020-05-13T10:26:57Z" duration-ms="4573" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Email=test@test.com, Special and offer code=CopperComplimentary, Address=JNTURoad, FirstName=Test, Enterzipcode=500085, Address2=NearMangeeraMall, EnterCurrency=USD, CallCenterUser=Test, roomtype=Times Square View Two Queen Room, Enterrooms=1 rooms, Cardname=test, Entercountry=India, Entercity=Hyderabad, Numberoncard=4321000000001119, Adults=1 adults, LastName=Test, Children=1 children, Enterstate=Telangana, Enterphonenum=1234567890}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:286)
	at org.testng.Assert.assertEquals(Assert.java:296)
	at Reusable_BE.TC_Mobile_New_Modify_CancelBooking.Flow(TC_Mobile_New_Modify_CancelBooking.java:212)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Flow -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:Reusable_BE.TC_Mobile_New_Modify_CancelBooking@10959ece]" started-at="2020-05-13T10:26:57Z" name="getResult" finished-at="2020-05-13T10:26:57Z" duration-ms="361" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Flow status=FAILURE method=TC_Mobile_New_Modify_CancelBooking.Flow(java.util.HashMap)[pri:0, instance:Reusable_BE.TC_Mobile_New_Modify_CancelBooking@10959ece] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method signature="Flow(java.util.HashMap)[pri:0, instance:Reusable_BE.TC_Mobile_New_Modify_CancelBooking@10959ece]" started-at="2020-05-13T10:26:57Z" name="Flow" description="Booking Room in Booking Engine" data-provider="MobileBooking" finished-at="2020-05-13T10:27:01Z" duration-ms="3535" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Email=test@test.com, Special and offer code=CopperComplimentary, Address=JNTURoad, FirstName=Test, Enterzipcode=500085, Address2=NearMangeeraMall, EnterCurrency=USD, CallCenterUser=Test, roomtype=Times Square View Two Queen Room, Enterrooms=1 rooms, Cardname=test, Entercountry=India, Entercity=Hyderabad, Numberoncard=4321000000001119, Adults=1 adults, LastName=Test, Children=1 children, Enterstate=Telangana, Enterphonenum=1234567890}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:286)
	at org.testng.Assert.assertEquals(Assert.java:296)
	at Reusable_BE.TC_Mobile_New_Modify_CancelBooking.Flow(TC_Mobile_New_Modify_CancelBooking.java:212)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1049)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1254)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Flow -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:Reusable_BE.TC_Mobile_New_Modify_CancelBooking@10959ece]" started-at="2020-05-13T10:27:01Z" name="getResult" finished-at="2020-05-13T10:27:01Z" duration-ms="356" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Flow status=FAILURE method=TC_Mobile_New_Modify_CancelBooking.Flow(java.util.HashMap)[pri:0, instance:Reusable_BE.TC_Mobile_New_Modify_CancelBooking@10959ece] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method is-config="true" signature="driverclose()[pri:0, instance:Reusable_BE.TC_Mobile_New_Modify_CancelBooking@10959ece]" started-at="2020-05-13T10:27:01Z" name="driverclose" finished-at="2020-05-13T10:27:02Z" duration-ms="1251" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- driverclose -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Reusable_BE.TC_Mobile_New_Modify_CancelBooking@10959ece]" started-at="2020-05-13T10:27:02Z" name="tearDown" finished-at="2020-05-13T10:27:03Z" duration-ms="689" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Reusable_BE.TC_Mobile_New_Modify_CancelBooking -->
    </test> <!-- tests -->
  </suite> <!-- preserve-order Suite -->
</testng-results>
